### PR TITLE
added auto focus to text control

### PR DESCRIPTION
**Pull request by AhmetTanakol**
reviewers: @ga97waqZ @doruktuncel @1903Daniel

### CODE REVIEW

**Comment by AhmetTanakol** [comment]
You don't need to merge, this is just some kind of a solution for autofocus

File: src/renderers/controls/text.control.tsx [hunk 1]
```diff
@@ -32,6 +32,7 @@ export class TextControl extends Control<ControlProps, void> {
                id={id}
                hidden={!visible}
                disabled={!enabled}
+               autofocus
```
Comment (1903Daniel) [inline src/renderers/controls/text.control.tsx]: 👍 all good

**Review by 1903Daniel — APPROVED** [review]
all good :)

**Comment by edgarmueller** [comment]
Great, this looks good! If you are worried about code repetition since you had to replicate the same code across all controls, don't be. We are aware that this is currently the case and we'll extract common props later.

**Pull request merged**